lib/ansible/modules/remote_management/manageiq/manageiq_user.py

```diff
@@ -183,7 +190,7 @@ def compare_user(self, user, name, group_id, password, email):
         """
         found_difference = (
             (name and user['name'] != name) or
-            (password is not None) or
+            (password is not None and self.module.params['update_password'] == 'always') or
```

Comment:
Is this enough?  IIUC this allows the module to skip POST if everything but password matches, but if something else is different (say port), it will stomp the password too.
I think you need logic in update to omit password from API request?

Reply:
a.
The issue is: we send update post request each time if "password" is set, because we can not compare it to current value.

This code fixes this issue.

b.
New issue: The name "update_password" implies that we do not update password, yes but, a name like "do_not_cheack_password_changes" is more confusing. current option is "just right" by me.

keep in mind the even is "update_password" is "on_creat", we do want to update the password is it's changed, or something else is changed, we just not want to send it always (see original issue) ...

c. Nice catch, we always update all fields and not just the changed ones, this is not specific to passwords, a code that send only the changed values will be nice (in a new PR)